packages/playground/data-liberation/src/import/WP_Entity_Importer.php

```diff
@@ -108,6 +113,9 @@ public function __construct( $options = array() ) {
 				'default_author'            => null,
 			)
 		);
+
+		WP_Topological_Sorter::activate();
+		$this->topological_sorter = new WP_Topological_Sorter( $this->options );
```

Comment:
It seems like we could only create this object in the topological sort stage, no? Right now it will stick around for the entire lifetime of `WP_Entity_Importer`.

Reply:
It is used even later during the import stage.